packages/xstate-wallet/src/workflows/conclude-channel.ts

```diff
@@ -19,6 +19,7 @@ const finalState = (store: Store) => async (context: Init): Promise<SupportState
 
   // If we've received a new final state that matches our outcome we support that
   if (outcomesEqual(latestSupportedByMe.outcome, latestFinalState?.outcome)) {
+    // outcomesEqual function will return false if there is inconsistent padding of hex strings
```

Comment:
What is this comment for? It's my understanding that `outcomesEqual` should _not_ return `false` in this case.

Reply:
The comment is telling you that the function does not do what you think it should.

Reply:
Shouldn't it be on `outcomesEqual` then?

Reply:
I propose that we do as you suggest and make this function behave independently of the formatting of the outcomes, say by using `bigNumberify`.

Reply:
(And then remove the comment)

Reply:
#1342